hudi-spark-datasource/hudi-spark-common/src/main/java/org/apache/hudi/internal/DataSourceInternalWriterHelper.java

```diff
@@ -105,4 +110,9 @@ public HoodieTable getHoodieTable() {
   public WriteOperationType getWriteOperationType() {
     return operationType;
   }
+
+  @VisibleForTesting
```

Comment:
is this annotation really needed?

Reply:
yes can we remove this. Not a great fan of informational annotations like these.